DeviceSetPoint[] set_points
--- 
bool success
string message
